modules/data.land/R/fia2ED.R

```diff
@@ -30,6 +30,7 @@ fia.to.psscss <- function(settings, gridres=0.075) {
 
 	## connect to database
 	con <-  db.open(settings$database$bety)
+	on.exit(db.close(con))
```

Comment:
that is cool (just as an FYI this is on exit of function, I had to look that up quickly).